src/pages/index.js

```diff
@@ -540,73 +580,116 @@ const Index = ({ data: fallbackData }) => {
+                      progress={Math.floor((tabCount / tabGoal) * 100)}
+                    />
+                  ) : null}
+                  {userGlobalId ? (
+                    <SearchForACauseSellModal
+                      userId={userGlobalId}
+                      hardSell={showSFACSellModalMode === 'hard-sell'}
+                      open={showSFACSellModalMode !== null}
+                      onAccept={onSFACSellModalAccept}
+                      onClose={() => setShowSFACSellModalMode(null)}
+                    />
+                  ) : null}
+                  {individualImpactEnabled ? (
+                    <UserImpactContainer
+                      userId={userGlobalId}
+                      userImpact={userImpact}
+                      user={user}
+                      disabled={
+                        missionsFeatureEnabled &&
+                        (missionStatus === 'started' ||
+                          missionStatus === 'completed')
+                      }
+                    />
+                  ) : (
+                    <Link to={aboutURL}>
+                      <IconButton>
+                        <InfoIcon
+                          className={clsx(
+                            classes.userMenuItem,
+                            classes.settingsIcon
+                          )}
+                        />
+                      </IconButton>
+                    </Link>
+                  )}
+                  <div className={classes.moneyRaisedContainer}>
+                    <Typography
+                      variant="h5"
+                      className={clsx(classes.userMenuItem)}
+                    >
+                      {growthbook.feature('test-feature').value ? (
+                        <p>Welcome to our site!</p>
+                      ) : null}
+                      <MoneyRaisedContainer app={app} user={user} />
+                    </Typography>
+                  </div>
+                  <div className={classes.settingsIconContainer}>
+                    <Link to={accountURL}>
+                      <IconButton>
+                        <SettingsIcon
+                          className={clsx(
+                            classes.userMenuItem,
+                            classes.settingsIcon
+                          )}
+                        />
+                      </IconButton>
+                    </Link>
+                  </div>
                 </div>
-                <div className={classes.settingsIconContainer}>
-                  <Link to={accountURL}>
-                    <IconButton>
-                      <SettingsIcon
-                        className={clsx(
-                          classes.userMenuItem,
-                          classes.settingsIcon
-                        )}
-                      />
-                    </IconButton>
-                  </Link>
+                {/**
+                 * TODO: consolidate all notifications here to manage
```

Comment:
What would be the ideal way to have another notification here? Asking for Search test. Just put it under and have it display below in the y-position?